test/javascripts/acceptance/discourse-reactions-notifications-test.js

```diff
@@ -241,3 +241,129 @@ acceptance("Discourse Reactions - Notifications", function (needs) {
+              user_id: 88,
+              notification_type: 25,
+              read: true,
+              high_priority: false,
+              created_at: "2022-08-18T13:00:11.166Z",
+              post_number: 12,
+              topic_id: 8432,
+              fancy_title: "Topic with one reaction from a user",
+              slug: "topic-with-one-reaction-from-a-user",
+              data: {
+                topic_title: "Topic with one reaction from a user",
+                original_post_id: 3349,
+                original_post_type: 1,
+                original_username: "krus",
+                revision_number: null,
+                display_username: "krus",
+                display_name: "Brucer Wayner II",
+                reaction_icon: "heart",
+                previous_notification_id: 933,
+                count: 1,
+              },
+            },
+            {
+              id: 842,
+              user_id: 88,
+              notification_type: 25,
+              read: true,
+              high_priority: false,
+              created_at: "2021-08-19T23:00:11.166Z",
+              post_number: 3,
+              topic_id: 138,
+              fancy_title: "Topic with 2 likes (total) from 2 users",
+              slug: "topic-with-2-likes-total-from-2-users",
+              data: {
+                topic_title: "Topic with 2 likes (total) from 2 users",
+                original_post_id: 443,
+                original_post_type: 1,
+                original_username: "jammed-radio",
+                revision_number: null,
+                display_username: "jammed-radio",
+                display_name: "Bruce Wayne I",
+                previous_notification_id: 933,
+                username2: "broken-radio",
+                name2: "Brucer Wayner II",
+                reaction_icon: "heart",
+                count: 2,
+              },
+            },
+            {
+              id: 2189,
+              user_id: 88,
+              notification_type: 25,
+              read: true,
+              high_priority: false,
+              created_at: "2020-11-13T03:10:41.166Z",
+              post_number: 31,
+              topic_id: 913,
+              fancy_title: "Topic with likes and reactions",
+              slug: "topic-with-likes-and-reactions",
+              data: {
+                topic_title: "Topic with likes and reactions",
+                original_post_id: 384,
+                original_post_type: 1,
+                original_username: "nuclear-reactor",
+                revision_number: null,
+                display_username: "nuclear-reactor",
+                display_name: "Monkey D. Luffy",
+                previous_notification_id: 933,
+                username2: "solar-engine",
+                name2: "Roronoa Zoro",
+                count: 4,
+              },
+            },
+          ],
+        });
+      });
+    });
+
+    test("reaction notifications with full name site setting on", async (assert) => {
```

Comment:
it's better to stick to the non-arrow functions as test methods (doesn't matter in this case, but if you were to use `this`, it wouldn't work)

```suggestion
    test("reaction notifications with full name site setting on", async function (assert) {
```